theme/templates/resource-landing-page/modals.html

```diff
@@ -1711,9 +1711,97 @@ <h4 class="modal-title" id="delete-funding-agency">Delete Funding Agency</h4>
             </div>
         </div>
     </div>
-
 {% endfor %}
 
+<div class="modal fade in" id="create-folder-dialog" tabindex="-1" role="dialog"
+     aria-labelledby="create-folder" aria-hidden="false">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+                <h4 class="modal-title" id="create-folder">New Folder</h4>
+            </div>
+
+            <div class="modal-body">
+                <div class="row">
+                    {# Folder Name #}
+                    <fieldset class="col-sm-12">
+                        <label>Folder Name *</label>
+                        <input id="txtFolderName" class="form-control modal-only-required">
+                    </fieldset>
+                </div>
+            </div>
+
+            <div class="modal-footer">
+                <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+                <button type="button" id="btn-create-folder" class="btn btn-primary disabled" data-dismiss="modal">Create
+                </button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="modal fade in" id="rename-dialog" tabindex="-1" role="dialog"
+     aria-labelledby="rename" aria-hidden="false">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+                <h4 class="modal-title" id="rename">Rename</h4>
+            </div>
+
+            <div class="modal-body">
+                <div class="row">
+                    <fieldset class="col-sm-12">
+                        <label>Name *</label>
+                        <div class="input-group">
+                            <input id="txtName" class="form-control modal-only-required">
+                            {% if cm|resource_type == "Time Series" or cm|resource_type == "Multidimensional (NetCDF)" %}
```

Comment:
@mauriel133 It looks like there is no "rename" action available for netCDF resource type, so this conditional check (at least for netCDF resource type) is not needed, right?